frontend/src/views/manager/CourseList.js

```diff
@@ -106,6 +108,17 @@ const CourseList = ({
   const conceptFilterParsed = search.parse(courseFilter)
   const includeCourse = course => search.include(course, conceptFilterParsed)
 
+  const openMenu = evt => setMenu({ anchor: evt.target, open: true })
+  const closeMenu = () => setMenu({ ...menu, open: false })
+  const selectMenu = tagName => () => {
+    let space = ' '
+    if (courseFilter.length === 0 || courseFilter.endsWith(' ')) {
+      space = ''
+    }
+    setCourseFilter(`${courseFilter}${space}tag:${tagName}`)
```

Comment:
Some filters have a space in it (e.g. "Data analysis"). The tag name could be surrounded with an apostrophe.